{"name": "登录用例", "status": "passed", "attachments": [{"name": "log", "source": "7e9fa47c-bf07-470d-94b8-326ddddd02ec-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "case_data", "value": "{'id': 'login_02', 'detail': '输入错误的账号', 'status': 'login_fail', 'data': {'account': '123456', 'pwd': '19971212hzj'}, 'check': ['用户名或密码不正确']}"}], "start": 1602673407018, "stop": 1602673412465, "uuid": "f49174b8-aed9-4e6f-9992-c18ac5ed3349", "historyId": "33a0672ddded07aae19c0949a2f5a833", "testCaseId": "3189d77ac603b701c4f11d350ad4e720", "fullName": "test_login.Test_CaseRun#test_search", "labels": [{"name": "feature", "value": "测试用例运行"}, {"name": "story", "value": "测试登录"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "Test_CaseRun"}, {"name": "host", "value": "WIN7-20190902OD"}, {"name": "thread", "value": "9472-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login"}]}